data/registry/instrumentation-js-fastify-otel.yml

```
@@ -0,0 +1,19 @@
+# cSpell:ignore fastify
+title: Fastify OpenTelemetry
+registryType: instrumentation
+language: js
+tags:
+  - Node.js
+  - fastify
+license: MIT
+description: Fastify OpenTelemetry auto-instrumentation library.
+authors:
+  - name: Fastify
+urls:
+  repo: https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-fastify
```

```suggestion
  repo: https://github.com/fastify/otel
```

shouldn't that point to the new repo?

---

Sorry, my bad; I re-use the one from the previous instrumentation